test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -130,9 +130,7 @@ private void TrainRegression(string trainDataPath, string testDataPath, string m
             // Potentially, the lines below can be in a different process altogether.
 
             // When you load the model, it's a 'dynamic' transformer. 
-            ITransformer loadedModel;
-            using (var stream = File.OpenRead(modelPath))
-                loadedModel = mlContext.Model.Load(stream);
+            ITransformer loadedModel = mlContext.Model.Load(modelPath);
```

Comment:
Just did this as a quick test. Should we add this as a separate test and include the option within the cookbook?